src/core/load-fragment.c

```diff
@@ -2390,6 +2390,375 @@ int config_parse_documentation(const char *unit,
 }
 
 #ifdef HAVE_SECCOMP
+static int syscall_filter_parse_one(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                ExecContext *c,
+                bool invert,
+                const char *t,
+                bool warn) {
+        int r;
+
+        if (*t == '@') {
+                const char * const *s;
+                static const char * const syscall_sets[] = {
+                        /* Default list */
+                        "@Default",
+                        "execve\0"
+                        "exit\0"
+                        "exit_group\0"
+                        "rt_sigreturn\0"
+                        "sigreturn\0",
+
+                        /* Need superuser capabilities */
+                        "@SysAdmin",
```

Comment:
why would you ever deny that explicitly? Isn't this more likely something you'd leave unlisted so that the blacklist blocks it?

Also, note that the syscalls you listed don't require privileges in all cases, for example the mount stuff is allowed if userns is used, which is unpriviliged.

Still not convinced there should be a group like this. Or can you name a really convincing example unit file utilizing this?